cms/envs/common.py

```diff
@@ -1127,3 +1130,12 @@
 
 # Partner support link for CMS footer
 PARTNER_SUPPORT_EMAIL = ''
+
+###################### Registration ##################################
+
+# For each of the fields, give one of the following values:
+# - 'required': to display the field, and make it mandatory
+# - 'optional': to display the field, and make it non-mandatory
+# - 'hidden': to not display the field
+
+REGISTRATION_EXTRA_FIELDS = {}
```

Comment:
In that case, this file should import this value from the lms/envs/common.py file.  Just add `REGISTRATION_EXTRA_FIELDS` to the `from lms.envs.common import` statement on the top of this file.